Sample.java

```diff
@@ -1,7 +1,39 @@
-// Time Complexity :
-// Space Complexity :
-// Did this code successfully run on Leetcode :
-// Any problem you faced while coding this :
 
 
 // Your code here along with comments explaining your approach
+/** 
+ * // Time Complexity : O(n*m)
+ * // Space Complexity : O(n*m)
```

Comment:
Define the variables please.

Reply:
Could you please explain what do you mean by "define the variables"?

Reply:
whats m and n